### ISSUE TITLE
[hi] Umbrella project: Localize "Contribute to K8s docs" subtopics to Hindi

### ISSUE THREAD

**Issue by rolfedh** [kubernetes/website#36455]
Umbrella project: Localize the subtopics in the "[Contribute to K8s docs](https://kubernetes.io/docs/contribute/)" section to Hindi.

Tasks:
1. Get a review/triage of this project.
2. If accepted, create subtasks for each topic.
3. Communicate tasks to the team.
  - Ask for volunteer contributors.
  - Ask that volunteers self-assign only one at one time. It should be complete before they self-assign another task.
4. Support progress to completion of the tasks.
5. Upon completion, close this project.

**Comment by rolfedh** [comment]
/hold
Hold until triaged.

**Comment by rolfedh** [comment]
Hi @bishal7679. I've updated the description to be clearer. These subtopics need to be localized to help other contributors join the project.
![image](https://user-images.githubusercontent.com/350907/187723842-6e35e8e1-8555-41ce-af55-81f0e9467b89.png)

**Comment by rolfedh** [comment]
@bishal7679 Yes. 
- You can create an issue for each topic. 
- In each issue, you can comment `Subtask for https://github.com/kubernetes/website/issues/36455`.
- You can also add a comment here with a list of the subtasks:
``` 
See the following subtasks:
- link to subtask 1
- link to subtask 2 
- etc..
```

The nice thing about these links is that they show the status of the issue.

Then, you can post each issue in #kubernetes-docs-hi and try to get one volunteer for each one.
Please guide folks to work on one issue at a time so that more contributors have an opportunity to contribute. (Fine to start a new one if they finish work on one and are waiting for merge.)

**Comment by amolmote** [comment]
That's great Idea, everyone will get a chance to contribute.@bishal7679  start raising issues, we will follow according to you. As you clarity of everything, take initiative of this , we will complete this as soon as possible.

**Comment by rolfedh** [comment]
Please add the "language/hi" label to these issues by commenting `/language hi` on each one.

**Comment by rolfedh** [comment]
I should have provided an example or template for the issues. The current one is for a different type of issue. Please use this issue as an example instead: https://github.com/kubernetes/website/issues/35939

**Comment by rolfedh** [comment]
Here's the raw text you can use as a template. Replace the filename string and 
```
# Prerequisites: 
- If this is your first contribution, visit [Contribute to K8s docs](https://kubernetes.io/docs/contribute/) for information on how to get started.
- Review a few [examples of localized glossary entries](https://github.com/kubernetes/website/issues/35600#issuecomment-1214215859) to see which content you change and which you live unchanged.

# Procedure:
1. Check out `dev-1.22-hi.1` and create a working branch called `localize-api-group`.
1. Create a file called `content/hi/docs/reference/glossary/api-group.md`.
1. Copy the content of https://raw.githubusercontent.com/kubernetes/website/main/content/en/docs/reference/glossary/api-group.md to the file you created.
1. Localize the content.
1. Push your change and create a pull request.

Completing this feature issue fixes part of https://github.com/kubernetes/website/issues/36455
```

**Comment by rolfedh** [comment]
Hi, @bishal7679. Thanks for making these updates. However, the old instructions must be removed. They do not make sense here.

**Comment by rolfedh** [comment]
Please ask new contributors to read the localization style guide that @AmanSarraf has made available here: https://github.com/kubernetes/website/pull/36133 (waiting for review and merge).

**Comment by amolmote** [comment]
I am done with my issue waiting for the one more review then will start to review the other's pr as well
